content/articles/integrating-chart.js-in-django/index.md

```diff
@@ -0,0 +1,305 @@
+
+```bash
+$ pip install django
+```
+
+Now, let's create a new Django project `Django-charts` using:
+
+```bash
+$ django-admin startproject twilio_proj
+```
+
+Then navigate to the `Django-charts` directory and create a Django app.
+
+```bash
+$ cd Django-charts
+$ django-admin startapp charts
+```
+
+Let's add our app to the list of installed apps.
+
+Navigate to the `Django-charts` directory and edit the `settings.py` file.
+
+```python
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'editors',
+]
+```
+
+### Creating a model
+We'll be creating a chart that shows the number of users for different text editors. So, for our model, we'll be creating just two fields. User field and text editor field.
+
+In the `editors` directory, edit the `models.py` file and add the following lines of code to it:
+
+```python
+from django.db import models
+
+#defining a simple class
+class Editors(models.Model):
+    editor_name = models.CharField(max_length=200)
+    num_users = models.IntegerField()
+
+    def __str__(self):
+        return "{}-{}".format(self.editor_name, self.num_users) 
+
+```
+
+Now let's register the model of Score in the `admin.py` file so that we can modify it in the Django admin section.
+
+```python
+from django.contrib import admin
+from .models import Editors
+
+admin.site.register(Editors)
+```
+
+The next step is to migrate our model to the database.
+
+Migrations are Django’s way of propagating changes you make to your models (adding a field, deleting a model, etc.) into your database schema.
+
+Now, let's migrate our model to the database by running the commands below:
+
+```bash
+$ python manage.py makemigrations # migrating the app and database changes
+$ python manage.py migrate        # final migrations
+```
+
+A superuser has the permissions to create, edit, update and delete data in Django admin. Create a superuser by running the command below:
+
+```bash
+$ python manage.py createsuperuser 
+```
+
+Now, we can log in to the admin page.
+
+To login to the admin section, serve the app again and go to http://localhost:8000/admin and log in with your superuser details.
+
+![Django admin login page](/engineering-education/content/articles/integrating-chart.js-in-django/login.png)
+
+*Django admin login page*
+
+![Django admin page](/engineering-education/content/articles/integrating-chart.js-in-django/admin.png)
+
+*Django admin page*
+
+Now, let's add an editor in the `Editors` category to test if its error-free.
+
+Click on editor, then click `Add Editor` to add an editor.
+
+![Add Editor](/engineering-education/content/articles/integrating-chart.js-in-django/add-editor.png)
+
+*Add editor*
+
+### Creating a view
+Now, let's create a view by adding the following to our `view.py` file:
```

Comment:
What's a view? Why do you need to create it? Brief a little about it.

(Beginners would find it difficult to understand)